Sources/BraintreeThreeDSecure/BTThreeDSecureV2Provider.swift

```diff
@@ -118,11 +116,24 @@ extension BTThreeDSecureV2Provider: CardinalValidationDelegate {
         stepUpValidated validateResponse: CardinalResponse!,
         serverJWT: String!
     ) {
-        let actionCodeString = analyticsString(for: validateResponse.actionCode)
-        apiClient.sendAnalyticsEvent("ios.three-d-secure.verification-flow.cardinal-sdk.action-code.\(actionCodeString)")
-
         switch validateResponse.actionCode {
-        case .success, .noAction, .failure:
+        case .success:
+            apiClient.sendAnalyticsEvent(BTThreeDSecureAnalytics.challengeSucceeded)
+            BTThreeDSecureAuthenticateJWT.authenticate(
+                jwt: serverJWT,
+                withAPIClient: apiClient,
+                forResult: lookupResult,
+                completion: completionHandler
+            )
+        case .failure:
+            apiClient.sendAnalyticsEvent(BTThreeDSecureAnalytics.challengeFailed)
+            BTThreeDSecureAuthenticateJWT.authenticate(
+                jwt: serverJWT,
+                withAPIClient: apiClient,
+                forResult: lookupResult,
+                completion: completionHandler
+            )
+        case .noAction:
             BTThreeDSecureAuthenticateJWT.authenticate(
```

Comment:
What is this case? Is it failure or success?

Reply:
Based on the above docs I listed digging into the website link it says the following for this status:
```
The API calls to Centinel API were completed and there is no further actionable items to complete. This can indicate that the card holder is not enrolled in 3-D Secure or it could indicate a validation error was encountered at Centinel Core. It is recommended that the rest of the response is reviewed to determine what has occurred.
```
It sounds like this could be a success or a failure which is... confusing at best. For the sake of analytics I wonder if we need another event for this since it doesn't seem to fit nicely anywhere. What do you think @KunJeongPark and @scannillo?

Reply:
I thought checking card holder is enrolled or not is done at `perform3DSLookup`, which is before getting to this point. 
 "validation error" means user's info is not right or some glitch happened to error out during validation?

Reply:
Let's just treat it as we were before, which is a "challenge success". [Android does the same](https://github.com/braintree/braintree_android/blob/6b0c5a60ad05a89771d23052f79f690848cf818a/ThreeDSecure/src/main/java/com/braintreepayments/api/ThreeDSecureClient.java#L507-L509), proceeds with the `/authenticate_from_jwt`